basket/news/tasks.py

```diff
@@ -286,21 +285,15 @@ def fxa_verified_sfdc(data):
         'fxa_lang': data.get('locale'),
         'fxa_service': data.get('service', ''),
         'fxa_id': fxa_id,
+        'optin': True,
+        'format': 'H',
     }
     if create_date:
         new_data['fxa_create_date'] = iso_format_unix_timestamp(create_date)
 
-    add_news = None
-    if newsletters:
-        if settings.FXA_REGISTER_NEWSLETTER not in newsletters:
-            newsletters.append(settings.FXA_REGISTER_NEWSLETTER)
-
-        add_news = ','.join(newsletters)
-    elif subscribe:
-        add_news = settings.FXA_REGISTER_NEWSLETTER
-
-    if add_news:
-        new_data['newsletters'] = add_news
+    newsletters = newsletters or []
+    newsletters.append(settings.FXA_REGISTER_NEWSLETTER)
```

Comment:
Just checking that it's OK to `append` if `FXA_REGISTER_NEWSLETTER` already exists since you have a check for this in previous revision. https://github.com/mozmeao/basket/pull/410/files#diff-239e0296d7bbbb9699bc1a16569f11b7L295

Reply:
Good catch. I did track down the code path after this and it does set math on the list, so it gets uniqueified later and so it didn't seem worth it to ensure uniqueness here as well.